overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -1504,9 +1504,17 @@
               "Damage:2": 5
             },
             "2:10": {
+              "id:8": "botania:livingwood",
+              "Count:3": 8
+            },
+            "3:10": {
               "id:8": "botania:pylon",
               "Count:3": 2,
               "Damage:2": 1
+            },
+            "4:10": {
+              "id:8": "botania:pool",
```

Comment:
given that in many cases mana pools are already placed down, this quest should not require picking them back up.